core/chains/evm/txmgr/orm.go

```diff
@@ -42,6 +42,78 @@ type orm struct {
 
 var _ ORM = (*orm)(nil)
 
+// Structs saved in database should be internal to
+// the ORM and distinct from generic params
+// so that modifying generic params does neccessitate
+// modification of underlying column types
+// (or runtime errors by mistake)
+type dbReceipt struct {
+	ID               int64
+	TxHash           common.Hash
+	BlockHash        common.Hash
+	BlockNumber      int64
+	TransactionIndex uint
+	Receipt          evmtypes.Receipt
+	CreatedAt        time.Time
+}
+
+type dbRawReceipt = evmtypes.Receipt
```

Comment:
note: this is is the definition of a raw receipt and is unlikely to be changed (even if we change the R type passed into `txmgrtypes.TxStorageService[common.Address, big.Int, common.Hash, NewTx, evmtypes.Receipt, EthTx, EthTxAttempt, int64, int64]`